scripts/update_mock_repo.py

```diff
@@ -44,24 +45,45 @@ def update_mock_repo():
     separately from
 
     """
-    target_file = join(
-        dirname(__file__), pardir, "tests", "data", "memote-mock-repo.tar.gz"
+    target_file = os.path.abspath(
+        join("tests", "data", "memote-mock-repo.tar.gz")
     )
     temp_dir = mkdtemp(prefix='tmp_mock')
+    previous_wd = os.getcwd()
     try:
         LOGGER.info("Cloning repository.")
-        Repo.clone_from(
-            'https://github.com/ChristianLieven/memote-mock-repo.git',
-            temp_dir)
+        os.chdir(temp_dir)
+        check_output(
+            ['git', 'clone',
+             'https://github.com/ChristianLieven/memote-mock-repo.git']
+        )
+        os.chdir('memote-mock-repo/')
+        LOGGER.info("Setting git to ignore filemode changes.")
+        call(
+            ['git', 'config',
+             'core.fileMode', 'false']
+        )
+        call(
+            ['git', 'config',
+             'user.email', 'memote@opencobra.com']
+        )
+        call(
+            ['git', 'config',
+             'user.name', 'memote-bot']
+        )
```

Comment:
If there was an exception here, shouldn't we log that message?

Reply:
Opened an issue for it!

https://github.com/opencobra/memote/issues/659